build.sh

```diff
@@ -7,24 +7,43 @@ tempdir=$(dirname $(mktemp -u))
 
 wintun_version="wintun-0.11"
 
-
 # until https://github.com/golang/go/issues/37475 is implemented
 VERSION=$(git describe --tags --always --abbrev=8 --dirty)
 
-
 # download dependencies
 download-wintun() {
   echo "check dependencies"
+
+  goos="$(go env GOOS)"
+  local force_download="$1"
+  if [ "$force_download" != "true" ] && [ "$goos" != "windows" ]; then
```

Comment:
Nice, feel that my previous scripts improvements should have included smth like that